app/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:14.04
+FROM ubuntu:17.04
```

Comment:
Good idea to upgrade, but I'll prefer to use LTS version, so `16.04` instead of `17.04`.